common/config.py

```diff
@@ -49,3 +49,10 @@
 DATE_TIME_FORMAT = "%Y-%m-%d %H:%M:%S"
 
 LAST_RUN_FILENAME = "job_last_run.json"
+
+DATADOG_OPTIONS = {
+    "statsd_host": "127.0.0.1",
+    "statsd_port": 8125,
```

Comment:
Maybe get this from the environment as well. While making 8125 the default if nothing is provided from the environment